codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/TestContextGenerator.kt

```diff
@@ -28,13 +28,13 @@ data class TestContext(
 
 class TestContextGenerator {
     companion object {
-        fun initContextFrom(modelFile: String, serviceShapeIdWithNamespace: String, protocol: ShapeId, rootPackageNamespace: String, sdkId: String): TestContext {
+        fun initContextFrom(modelFile: String, serviceShapeIdWithNamespace: String, protocol: ShapeId): TestContext {
```

Comment:
`rootPackageName` was removed as a param because it was unused.
`sdkId` was removed as a param because it was unused, and because it can be retrieved from the service instead.